ComplexQueriesAndProcedures/AddNewBorrowers.sql

```diff
@@ -0,0 +1,21 @@
+CREATE PROCEDURE AddNewBorrower
```

Comment:
1. the stored procedure name should follow the name on the task
2. and always keep the file name as storedProcedure/function name